JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;CA2007;CA1054;CA1307;CA1063;CA1822;CA1052;CA1034;CA1816</NoWarn>

```

Comment:
Should prefix with `$(NoWarn);`, otherwise you'll overwrite any warnings configured to be ignored elsewhere.